app/decorators/service_provider_session_decorator.rb

```diff
@@ -13,6 +13,14 @@ class ServiceProviderSessionDecorator
       i18n_name: 'usa_jobs',
       learn_more: 'https://login.gov/help/',
     },
+    'Railroad Retirement Board' => {
+      i18n_name: 'railroad_retirement_board',
+      learn_more: 'https://login.gov/help/',
+    },
+    'U.S. Railroad Retirement Board Benefit Connect' => {
+      i18n_name: 'railroad_retirement_board',
+      learn_more: 'https://login.gov/help/',
+    },
```

Comment:
Do we need to come up with a better solution than this? Perhaps the `learn_more` or other contact info should be part of the SP configuration?

Reply:
Agree. This was supposed to be a one-off for TTP that has grown into something a bit more complicated.